lib/tasks/backfill_constituent_grades.rake

```diff
@@ -3,10 +3,18 @@ task backfill_constituent_grades: :environment do
   structured_qualifications = ApplicationQualification.where(constituent_grades: nil).where.not(structured_grades: nil)
 
   structured_qualifications.each do |qualification|
-    grades = JSON.parse(qualification.structured_grades)
+    grades = get_structured_grades_hash(qualification)
 
-    constituent_grades = grades.transform_values { |grade| { grade: grade } }.to_json
```

Comment:
This line now means we store hashes and not JSON strings.
So we don't have to parse it when reading, and don't have to convert to json when writing.

(This will require me to change how constituent_grades are read/written, but that's all in a future PR anyway)